src/odemis/driver/piezomotor.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, name, role, port, axes, inverted=None, param_file=None, **kwa
         """
         self._axis_map = {}  # axis name -> axis number used by controller
         self._closed_loop = {}  # axis name (str) -> bool (True if closed loop)
+        self._speed = {}  # axis name (str) -> speed in unit/s
```

Comment:
Isn't this always [m/s]?

Reply:
There is a parameter to specify the unit (defaults to "m"). So the speed will be in this unit. Admittedly, as in practice so far it's always in "m", I've left the .speed unit to be "m/s", but it's not formally correct. (just as incorrect as in the other drivers) ;-)